pages/404.tsx

```diff
@@ -0,0 +1,92 @@
+import Image from 'next/image';
+import type { NextPage } from 'next';
+import Link from 'next/link';
+import SEO from 'components/site/seo';
+import LayoutWrap from 'components/site/layoutWrap';
+import LayoutInnerContent from 'components/site/layoutInnerContent';
+import { Box } from '@sparkpost/matchbox';
+import styled from 'styled-components';
+import css from '@styled-system/css';
+
+const StyledLink = styled(Box)`
+  &,
+  &:visited {
+    ${css({
+      color: 'gray.900',
+    })}
+  }
+
+  &:hover {
+    ${css({
+      color: 'blue.700',
+    })}
+  }
+`;
+
+const OhhhNoes: NextPage = () => {
+  return (
+    <>
+      <SEO
+        title={'404 Page'}
+        description={'Oops! Looks like we can’t find the page you’re looking for.'}
+      />
+      <LayoutWrap bg="white">
+        <LayoutInnerContent>
+          <Box display="flex" flexDirection="column" alignItems="center" width="100%">
+            <Box
+              as="h1"
+              fontSize={['800', null, '228.39px']}
+              lineHeight={['800', null, '285.49px']}
+              fontWeight="semibold"
+              mt={['0', null, '750']}
+            >
+              404
```

Comment:
There was some concern on the UX team about this page expanding past the fold. Could we adjust everything to be fluid to make sure the image is within 100 vh?